main.js

```diff
@@ -108,10 +118,11 @@ function createFormContainer(){
     return divFormContainer;
 }
 
-function createInputField(divId, inputName, labelText){
+function createInputField(divId, paragraphId, inputName, labelText, {inputValue, error}){
```

Comment:
`function createInputField(divId, paragraphId, inputName, labelText, {inputValue, error}){`

Let's keep the same style of arguments.

Just make all arguments an object or an array, like:

`function createInputField({ divId, paragraphId, inputName, labelText, inputValue, error }){`
or
`function createInputField(divId, paragraphId, inputName, labelText, inputValue, error){`

Reply:
Done